models/document/src/index.ts

```diff
@@ -0,0 +1,176 @@
+//
+// Copyright © 2020, 2021 Anticrm Platform Contributors.
```

Comment:
We need way to fix it everywhere (or probably do it manually)

Reply:
Fixed.